src/styles.module.pcss

```diff
@@ -0,0 +1,211 @@
+.nft-tool {
+  margin: 1em auto;
+  display: flex;
+  flex-direction: row;
+  /* flex-direction: column; */
+
+  align-items: stretch;
+  padding: 20px 0;
+  gap: 30px;
+
+  &__form {
+    width: 100%;
+
+    &-param:not(:last-child) {
+      margin-bottom: 16px;
+    }
+
+    &-label {
+      display: block;
+      font-weight: bold;
+      font-size: 14px;
+      padding-bottom: 6px;
+    }
+
+    &-input {
+      width: 100%;
+      padding: 8px 12px;
+      border-radius: 4px;
+      border: 1px solid #EFF0F1;
+      background-color: #F8F8F8;
+      font-size: 14px;
+      line-height: 1.4;
+      color: #000;
+      outline: none;
+      transition: border-color 0.2s ease-in-out;
+
+      &:focus {
+        border-color: #007aff;
+      }
+    }
+
+    &-select {
+      width: 100%;
+      padding: 8px 12px;
+      border-radius: 4px;
+      border: 1px solid #EFF0F1;
+      background-color: #F8F8F8;
+      font-size: 14px;
+      line-height: 1.4;
+      color: #000;
+      outline: none;
+
+
+      background-repeat: no-repeat;
+      background-position: right 10px bottom 50%;
+
+      cursor: pointer;
+
+      appearance: none;
+      -moz-appearance: none;
+      -webkit-appearance: none;
+
+      &:focus {
+        border-color: #007aff;
+      }
+    }
+
+    &-button {
+      width: 100%;
+      padding: 8px 12px;
+      border-radius: 4px;
+      border: 1px solid #EFF0F1;
+      background-color: #fff;
+      font-size: 14px;
+      line-height: 1.4;
+      color: #333;
+      outline: none;
+      transition: border-color 0.2s ease-in-out;
+
+      &:hover {
+        cursor: pointer;
+        background-color: rgba(0, 0, 0, 0.05);
+      }
+
+      &:active {
+        background-color: rgba(0, 0, 0, 0.1);
+      }
+
+      &:focus {
+        border-color: #007aff;
+      }
+    }
+  }
+
+  &__card {
+    display: flex;
+    flex-direction: column;
+    align-items: column;
+    padding: 40px;
+
+    width: 230px;
+
+    border-radius: 16px;
+    border: 1px solid #EFF0F1;
+    box-shadow: 0px 2px 3px rgba(0, 0, 0, 0.03);
+
+    background: #FFFFFF;
+
+    &-image {
+      border-radius: 16px;
+      overflow: hidden;
+
+      width: 100%;
+
+      background-color: #f0f0f0;
+
+      &::-webkit-scrollbar {
+        display: none;
+      }
+    }
+
+    &-title {
+      margin-top: 20px;
+
+      font-style: normal;
+      font-weight: bold;
+      font-size: 16px;
+      line-height: 23px;
+
+      text-align: center;
+    }
+
+    &-collection {
+      margin-top: 4px;
+
+      font-weight: regular;
+      font-size: 14px;
+      line-height: 22px;
+      text-align: center;
+
+      color: #707684;
+    }
+  }
+}
+
+.content {
```

Comment:
will such simple class be replaced with hash of CSS Modules? if so, you can use such simple classes instead of BEM in a code above